### PR TITLE
Modified homepage

### PR DESCRIPTION
**Pull request by sanketbansal**
#### Changes proposed in this pull request:

- Modified the home component to make its UI similar to current EvalAI UI.

- Link to live demo: http://pr-140-evalai.surge.sh 

Screenshot from 2019-06-05 22-01-50
-
Screenshot from 2019-06-05 22-01-58
-
Screenshot from 2019-06-05 22-02-05
-
Screenshot from 2019-06-05 22-02-20

### CODE REVIEW

**Comment by lunayach** [comment]
@sanketbansal Could you see why the tests are failing?

**Comment by sanketbansal** [comment]
Working on it

**Comment by lunayach** [comment]
@sanketbansal again 137 file changes! Are all of them for this PR only?

**Comment by sanketbansal** [comment]
@lunayach No! it includes file changes of static styles and assets for which I have created a separate PR #154, #156, #157.
Please review those PRs first then I will rebase it.

**Comment by lunayach** [comment]
@sanketbansal Yes, I have already reviewed #154. Please see.

**Comment by sanketbansal** [comment]
@RishabhJain2018 @lunayach @Shekharrajak @galipremsagar please review

**Comment by lunayach** [comment]
205 file changes! Need to review/merge #154, I guess.

**Comment by sanketbansal** [comment]
@lunayach file changes has been reduced please review

**Comment by lunayach** [comment]
@sanketbansal  I see, that it still has `styles-port` folder?

**Comment by sanketbansal** [comment]
Yeah I just kept it to show you the new base styles which needs to be add for the whole project as discussed. I will remove it after your review.

**Comment by lunayach** [comment]
After resolving #141, @sanketbansal resolve conflicts here.

File: src/app/components/nav/footer/footer.component.html [hunk 1]
```diff
@@ -1,61 +1,40 @@
...
-		<div class="footer-jump-links top-margin-extra">
-		<a target="_blank" href="https://github.com/Cloud-CV"><img src="assets/images/git.png" /></a>
-		<a target="_blank" href="https://twitter.com/project_cloudcv"><img src="assets/images/twitter.png" /></a></div>
-	</div>
-	</div>
-	</div>
-</div>
-</div>
+<footer class="ev-md-container ev-footer ev-dark-bg">
+  <div class="grad-container rm-grad-pad">
```
Comment (lunayach) [inline src/app/components/nav/footer/footer.component.html]: Same changes as #141. Let's merge #141 first.

**Comment by lunayach** [comment]
@sanketbansal large no. of conflicts are for this one; please resolve them.

File: src/app/components/home/home.component.html [hunk 2]
```diff
@@ -1,16 +1,234 @@
...
+            </a>
+          </div>
+          <div *ngIf="challengeList.length == 0" class="row">
+            <p class="text-dark-black w-300 center fs-16">Sorry! there are no featured challenges. Please check again later.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- organizatons section -->
+    <section class="ev-container ev-reverse-details text-med-black ev-super-light-bg " id="ev-scroll-down">
```
Comment (lunayach) [inline src/app/components/home/home.component.html]: Keep the ID's unique. This, https://github.com/Cloud-CV/EvalAI-ngx/pull/140/files#diff-52b8edd82669758b98abb08f9c2a93b1R212, and https://github.com/Cloud-CV/EvalAI-ngx/pull/140/files#diff-52b8edd82669758b98abb08f9c2a93b1R26 have the same ID's.

File: src/app/components/home/home.component.spec.ts [hunk 3]
```diff
@@ -60,17 +60,4 @@ describe('HomeComponent', () => {
   it('should be created', () => {
     expect(component).toBeTruthy();
   });
-
```
Comment (lunayach) [inline src/app/components/home/home.component.spec.ts]: I see why you have removed the test specs here. But add to **TODO's** to add new test cases here.

File: src/app/components/home/home.component.ts [hunk 4]
```diff
@@ -8,17 +14,50 @@ import { Component, OnInit } from '@angular/core';
...
+      () => {}
+    );
+
+  }
+
+  hostChallenge() {
+    if (this.authService.isAuth) {
+      this.router.navigate(['/challenge-create']);
+    } else {
+      this.router.navigate(['/auth/login']);
```
Comment (lunayach) [inline src/app/components/home/home.component.ts]: Don't hard-code the URL's; like in previous PR's, define them as constants above.

File: src/app/components/home/home.component.ts [hunk 5]
```diff
@@ -8,17 +14,50 @@ import { Component, OnInit } from '@angular/core';
...
+    this.init();
+    this.getChallenge();
+  }
 
-  /**
-   * Component on initialized.
-   */
-  ngOnInit() {}
+  init() {
+    const logInStatus = this.authService.isLoggedIn();
```
Comment (lunayach) [inline src/app/components/home/home.component.ts]: Where are we using `logInStatus`?

File: src/app/services/auth.service.ts [hunk 6]
```diff
@@ -16,8 +16,14 @@ export class AuthService {
    */
   isAuth = false;
 
+  /**
+   * Porting Ends
```
Comment (lunayach) [inline src/app/services/auth.service.ts]: Remove this.

File: src/styles/base.scss [hunk 7]
```diff
@@ -392,6 +393,10 @@ mat-chip {
   padding: 30px 40px 40px 40px;
 }
 
+//.ev-md-container {
```
Comment (lunayach) [inline src/styles/base.scss]: Remove this.

File: src/styles/base.scss [hunk 8]
```diff
@@ -1452,8 +1457,20 @@ td, th {
...
+  background-color: $highlight;
+  i {
+    font-size: 16px;
+  }
+}
+
+.btn-pagination:hover {
+  background-color: $highlight;
+  i {
+    font-size: 16px;
```
Comment (lunayach) [inline src/styles/base.scss]:
```suggestion
    font-size: $fs-16;
```
Also, above.

**Review by lunayach — COMMENTED** [review]
Functionality works! Minor changes needed.

File: src/styles/base.scss (hunk 8 above)
Comment (sanketbansal) [reply src/styles/base.scss]: done

File: src/styles/base.scss (hunk 7 above)
Comment (sanketbansal) [reply src/styles/base.scss]: done

File: src/app/services/auth.service.ts (hunk 6 above)
Comment (sanketbansal) [reply src/app/services/auth.service.ts]: done

File: src/app/components/home/home.component.ts (hunk 5 above)
Comment (sanketbansal) [reply src/app/components/home/home.component.ts]: done

File: src/app/components/home/home.component.ts (hunk 4 above)
Comment (sanketbansal) [reply src/app/components/home/home.component.ts]: done

File: src/app/components/home/home.component.spec.ts (hunk 3 above)
Comment (sanketbansal) [reply src/app/components/home/home.component.spec.ts]: done

File: src/app/components/home/home.component.html (hunk 2 above)
Comment (sanketbansal) [reply src/app/components/home/home.component.html]: done

**Review by lunayach — APPROVED** [review]
LGTM! @RishabhJain2018, Homepage is smooth now.

**Pull request merged**